.github/workflows/build.yml

```diff
@@ -7,6 +7,10 @@ on:
     branches:
       - "*"
 
+concurrency:
```

Comment:
This will abort previous builds if new commits come in faster than it can build.